testhelpers/org.eclipse.n4js.tester.tests.helper/META-INF/MANIFEST.MF

```diff
@@ -13,7 +13,7 @@ Bundle-ClassPath: commons-codec-1.6.jar,
  hamcrest-library-1.3.jar,
  httpclient-4.5.2.jar,
  httpcore-4.4.6.jar,
- httpmime-4.3.6.jar,
+ httpmime-4.5.2.jar,
```

Comment:
This PR changes the version number of httpmime but does not change the corresponding .jar file in root folder of bundle `org.eclipse.n4js.tester.tests.helper`. Is this correct??